Objects/listobject.c

```diff
@@ -2760,6 +2761,17 @@ static PyMethodDef listiter_methods[] = {
     {NULL,              NULL}           /* sentinel */
 };
 
+static PySequenceMethods listiter_as_sequence = {
+    0,                                          /* sq_length */
+    0,                                          /* sq_concat */
+    0,                                          /* sq_repeat */
+    0,                                          /* sq_item */
+    0,                                          /* sq_slice */
+    0,                                          /* sq_ass_item */
+    0,                                          /* sq_ass_slice */
+    (objobjproc)listiter_contains,              /* sq_contains */
```

Comment:
CPython now allows the C99 syntax. The initializer of this structure can be written as
```
static PySequenceMethods listiter_as_sequence = {
    .sq_contains = (objobjproc)listiter_contains,
};
```
This would decrease the size of the code, but not too much.